stripe/src/main/java/com/stripe/android/view/CardInputWidget.java

```diff
@@ -120,9 +120,10 @@ public Card getCard() {
         }
 
         // CVC/CVV is the only field not validated by the entry control itself, so we check here.
-        int requiredLength = mIsAmEx ? Card.CVC_LENGTH_AMERICAN_EXPRESS : Card.CVC_LENGTH_COMMON;
-        String cvcValue = mCvcNumberEditText.getText().toString();
-        if (StripeTextUtils.isBlank(cvcValue) || cvcValue.length() != requiredLength) {
+        String cvcValue = mCvcNumberEditText.getText().toString().trim();
+        int cvcLength = cvcValue.length();
+        if (!(cvcLength == Card.CVC_LENGTH_COMMON
```

Comment:
can we simplify this expression at all? I'm worried it'll be confusing down the line with all the nesting and the parts

One idea is we can either pull it into two functions 
isCommonCSVLength() || isValidAmexCVC()